ehrapy/api/anndata_ext.py

```diff
@@ -102,6 +102,50 @@ def anndata_to_df(adata: AnnData, add_from_obs: list[str] | str | None = None) -
     return df
 
 
+def move_to_obs(adata: AnnData, to_obs: list[str] | str) -> None:
+    """Move some columns from X to obs inplace
+    Args:
+        adata: The AnnData object
+        to_obs: The columns to move to obs
+
+    Returns:
+        The original AnnData object with moved columns from X to obs
+    """
+    if isinstance(to_obs, str):
+        to_obs = [to_obs]
+    # don't allow moving encoded columns as this could lead to inconsistent data in X and obs
+    if any(column.startswith("ehrapycat") for column in to_obs):
+        raise ObsMoveError("Cannot move encoded columns from X to obs. Either undo encoding or remove them from the list!")
```

Comment:
Ain't no undo encoding yet in ehrapy :)
But guess the instructions are still fine.

Reply:
We do ;). It just resets thew whole anndata object though.